src/biocommons/seqrepo/seqaliasdb/seqaliasdb.py

```diff
@@ -32,7 +32,6 @@ def __init__(
         check_same_thread=True,
     ):
         self._db_path = db_path
-        self._db = None
```

Comment:
not sure what this was doing here. As far as I can tell, a) nothing would work if it stays `None`, and b) it is impossible to not reassign this property to an actual DB connection, so there's zero reason to leave it as possibly `None`.

Reply:
Might be good to separate this out in a separate PR since it's not related to current PR title